lib/gollum/app.rb

```diff
@@ -227,9 +227,11 @@ class App < Sinatra::Base
           options.merge! author
         end
 
+        normalize = !(Gollum::File.image?(fullname) || Gollum::File.binary?(fullname))
```

Comment:
You could dispense with the new `normalize` variable and just add the `!(...)` to the hash below, but if you think this is more readable, 👍

Reply:
Okay, in that case I'd like to keep it as is for readability. 👍